firestore-emulator/typescript-quickstart/test/test.ts

```diff
@@ -0,0 +1,67 @@
+// Reference mocha-typescript's global definitions:
```

Comment:
Can we sanitize this file a bit? Change the names and values.

Reply:
For sure. I just wrote a bug against me.